src/PythonMigrationViewExtension/Controls/VisualDifferenceViewer.xaml

```diff
@@ -0,0 +1,107 @@
+﻿<Window x:Class="Dynamo.PythonMigration.Controls.VisualDifferenceViewer"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:p="clr-namespace:Dynamo.PythonMigration.Properties;assembly=PythonMigrationViewExtension"
+        xmlns:fa="http://schemas.fontawesome.io/icons/"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        xmlns:diffplex="clr-namespace:DiffPlex.Wpf.Controls;assembly=DiffPlex.Wpf"
+        mc:Ignorable="d"
+        d:DesignHeight="450"
+        d:DesignWidth="800"
+        Title="Migration Assistant"
+        WindowStartupLocation="CenterOwner"
+        Background="#272822">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="44" />
+        </Grid.RowDefinitions>
+
+        <Grid.Resources>
+            <Style TargetType="Button"
+                   BasedOn="{StaticResource SButtonWithShapeIcon}">
+                <Setter Property="Margin"
+                        Value="1" />
+                <Setter Property="VerticalAlignment"
+                        Value="Bottom" />
+            </Style>
+
+        </Grid.Resources>
+
+        <diffplex:DiffViewer x:Name="DiffView"
+                             OldTextHeader="Old"
+                             NewTextHeader="New"
+                             FontFamily="Consolas"
+                             FontSize="10pt"
+                             Background="#272822"
+                             Foreground="#F8F8F2"
+                             HeaderForeground="#808080">
+        </diffplex:DiffViewer>
```

Comment:
sorry @mmisol not sure i understand about the minimum resolution part?

It should only fit the code in the editor once pr view, if you change the view to it will refit the code to the window. Using text wrap is not an easy fix as the `DiffViewer` uses ScrollViewers for the content, i guess wrapping the text in scrollviewer isnt really possible and might not make sense. Let me know if you guys want to look into wrapping the text instead of using scrollviewers.

When making the window smaller the intent was that horizontal scrollbars should appear, i guess that this is the question of whether we want to wrap the text or have scrollbars.

Reply:
Preference would be to match the existing Dynamo experience with wrapping - it's just a matter of effort. I agree @mmisol that if it's hard to do then we don't, but @SHKnudsen please have a look and see if it's an easy win.